matrix_client/api.py

```diff
@@ -377,19 +377,15 @@ def media_upload(self, content, content_type):
 
     def get_display_name(self, user_id):
         content = self._send("GET", "/profile/%s/displayname" % user_id)
-        if "displayname" not in content.keys():
-            return user_id
-        return content['displayname']
+        return content.get('displayname', user_id)
```

Comment:
OK. Any suggestions for what the helper method should be called? Or should we leave this as is, and have a separate 'is_displayname_set' method, or something like that?